package.json

```diff
@@ -93,6 +95,7 @@
   },
   "homepage": "https://github.com/cloudevents/sdk-javascript#readme",
   "dependencies": {
+    "@types/node": "^13.13.5",
```

Comment:
All `@types` modules should not be included in a dependency. It has no runtime functionality.

Reply:
I think this should be a `devDependency` since `tsc` needs it when type checking.

Reply:
Yes, it's always a dev dep.